app/templates/components/event-table.hbs

```diff
@@ -0,0 +1,84 @@
+  <div class="row">
+    <div class="sixteen wide column">
+      <br>
+      <table class="ui tablet stackable celled table">
+        <thead>
+          <tr>
+            <th>Logo</th>
+            <th>Name</th>
+            <th>Date</th>
+            <th>Roles</th>
+            <th>Sessions</th>
+            <th>Speakers</th>
+            <th>Tickets</th>
+            <th>Public Url</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <div>
+                <img src="http://placehold.it/100x100" alt="Event logo">
+              </div>
+            </td>
+            <td>
+              <div>Sample Event</div>
+            </td>
+            <td>
+              March 18, 2016 - 09:30 AM
+              <br>(to)<br>
+              March 20, 2016 - 05:30 PM
+            </td>
+            <td>
+              <div class="ui ordered list">
+                <div class="item">sample@gmail.com (Organizer)</div>
+                <div class="item">sample2@gmail.com (Manager)</div>
+              </div>
+            </td>
+            <td>
+              <div class="ui list">
+                <div class="item">Drafts: 0</div>
+                <div class="item">Submitted: 0</div>
+                <div class="item">Accepted: 0</div>
+                <div class="item">Confirmed: 0</div>
+                <div class="item">Pending: 0</div>
+                <div class="item">Rejected: 0</div>
+              </div>
+            </td>
+            <td class = "center aligned">
+              2
+            </td>
+            <td>
+              <div class="ui bulleted list">
+                <div class="item">Premium (12/100)</div>
+                <div class="item">VIP (10/15)</div>
+                <div class="item">Normal (100/200)</div>
+                <div class="item">Free (100/500)</div>
+              </div>
+            </td>
+            <td>
+              <div class="ui link list">
```

Comment:
to give it a uniform look with the UI, instead of the blue coloured default link